classes/form/edit_entry.php

```diff
@@ -27,6 +27,21 @@ public function definition() {
         $mform->addElement('text', 'name', get_string('entryname', 'mod_cpdlogbook'));
         $mform->setType('name', PARAM_TEXT);
```

Comment:
The one field which is missing is the date that the entry happened on. There is currently a 'time' field but its not clear if this was intended as a timestamp for the creation of the entry of for when the entry happened. Later it probably needs to be split into two fields. For now lets assume time is the date when the training happened not the creation timestamp. It can use a date_selector and probably only needs to be a date rather than a date time. For clarity I'd probably rename this field to something like 'completiondate'